CloudFabric.EventSourcing.Common/Extensions/ServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,64 @@
+using CloudFabric.EventSourcing.EventStore.Postgresql;
+using CloudFabric.Projections;
+using CloudFabric.Projections.Postgresql;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace CloudFabric.EventSourcing.Common.Extensions
+{
+    public static class ServiceCollectionExtensions
+    {
+        public static IEventSourcingBuilder AddPostgresqlEventStore(
```

Comment:
We will need to have same for Postgres, InMemory, CosmosDb etc. Maybe all of this should be part of implementation-specific nuget packages?